importer/api/__init__.py

```diff
@@ -0,0 +1,95 @@
+"""
+Import OLX data into LORE.
+"""
+
+from glob import glob
+from datetime import datetime
+from shutil import rmtree
+from tempfile import mkdtemp
+from os.path import join
+import tarfile
+import zipfile
+
+from xbundle import XBundle, DESCRIPTOR_TAGS
+from lxml import etree
+
+from learningobjects.api import create_course, create_lox
+
+
+def import_course_from_file(filename, user_id):
+    """
+    Import OLX from .zip or tar.gz.
+
+    Args:
+        filename (unicode): Path to archive file (zip or .tar.gz)
+    Raises:
+        ValueError: Unable to find single path inside archive file.
+    Returns: None
+    """
+    tempdir = mkdtemp()
+    if filename.endswith(".tar.gz"):
+        course = tarfile.open(filename, "r")
+    elif filename.endswith(".zip"):
+        course = zipfile.ZipFile(filename, "r")
```

Comment:
It does, and works well. However, not in Python 3 because it uses `basestring` to check if the filename passed is a string. It should be a simple fix and I'd like to fork and do a PR to the original package author. Should I fork it under `mitodl`, or just my own account?